esignet-integration-api/src/main/java/io/mosip/esignet/api/dto/VCResult.java

```diff
@@ -0,0 +1,18 @@
+package io.mosip.esignet.api.dto;
+
+import lombok.Data;
+
+@Data
+public class VCResult<T> {
+
+    /**
+     * Format of credential
+     * Eg: ldp_vc
+     */
+    private String format;
+
+    /**
+     *
+     */
+    private T credential;
```

Comment:
where is this T feeded from ?

Reply:
Datatype conformation is in VCIssuanceServiceImpl